### PR TITLE
unittests

### PR DESCRIPTION
**Pull request by aryamccarthy**
I added two unit tests.

### CODE REVIEW

File: littlebird/tweet_tokenizer.py [hunk 1]
```diff
@@ -67,18 +62,21 @@ def __init__(self,
...
         self.HASHTAG_RE = regex.compile(r"#[\p{L}\p{N}_]+")
-        self.REMOVAL_RE = regex.compile("|".join([self.HANDLE_RE, self.URL_RE, self.RT_RE]))
+        self.REMOVAL_RE = regex.compile(
+            "|".join([self.HANDLE_RE, self.URL_RE, self.RT_RE])
+        )
         self.WHITESPACE_RE = regex.compile(r"\s+")
         self.TOKEN_RE = regex.compile(token_pattern)
         self.remove_hashtags = remove_hashtags
         self.lowercase = lowercase
+        self.stopwords: Optional[Set[str]]
```
Comment (AADeLucia) [inline littlebird/tweet_tokenizer.py:72]: what is the point of this line if this variable is being initialized a few lines down? Does this actually initialize anything or just declare the type?

File: littlebird/tweet_tokenizer.py [hunk 2]
```diff
@@ -123,28 +122,36 @@ def tokenize_tweet_file(self, input_file, sample_size=-1, return_tokens=False):
...
-        
+
         # Tokenize the tweets and return
         # Some tweets have no valid tokens. Skip them.
-        tweet_text = map(self.tokenize, all_tweet_text)
+        tweet_text_ = map(self.tokenize, all_tweet_text)
+        tweet_text: Union[List[str], List[List[str]]]
         if return_tokens:
-            tweet_text = [t for t in tweet_text if t != []]
+            tweet_text = [t for t in tweet_text_ if t != []]
```
Comment (AADeLucia) [inline littlebird/tweet_tokenizer.py:137]: Intermediate variable?

File: setup.py [hunk 3]
```diff
@@ -18,16 +18,12 @@
     long_description_content_type="text/markdown",
     url="",
     packages=setuptools.find_packages(),
-    install_requires=[
-        "jsonlines",
-        "regex",
-        "filetype",
-    ],
+    install_requires=["jsonlines", "regex", "filetype"],
```
Comment (AADeLucia) [inline setup.py:21]: Sometimes you use multiple lines, other times you use a single line. What's your reasoning?

File: littlebird/tweet_tokenizer.py (hunk 1 above)
Comment (aryamccarthy) [reply littlebird/tweet_tokenizer.py:72]: Just declares the type

File: littlebird/tweet_tokenizer.py (hunk 2 above)
Comment (aryamccarthy) [reply littlebird/tweet_tokenizer.py:137]: So the type of each label doesn't change. Suppose it's not necessary in hindsight

File: setup.py (hunk 3 above)
Comment (aryamccarthy) [reply setup.py:21]: Auto-formatted with [black](https://github.com/psf/black)

**Pull request merged**